examples/functions/trip/TR_ParallelService_OeBB_wrong.xml

```diff
@@ -0,0 +1,243 @@
+									<StopPointName>
+										<Text xml:lang="de">Wels</Text>
+									</StopPointName>
+									<ServiceArrival>
+										<TimetabledTime>2023-08-15T21:48:00Z</TimetabledTime>
+									</ServiceArrival>
+									<ServiceDeparture>
+										<TimetabledTime>2023-08-15T21:50:00Z</TimetabledTime>
+									</ServiceDeparture>
+									<Order>4</Order>
+								</LegIntermediate>
+								<LegIntermediate>
+									<siri:StopPointRef>5</siri:StopPointRef>
+									<StopPointName>
+										<Text xml:lang="de">Passau Hbf</Text>
+									</StopPointName>
+									<ServiceArrival>
+										<TimetabledTime>2023-08-15T22:35:00Z</TimetabledTime>
+									</ServiceArrival>
+									<ServiceDeparture>
+										<TimetabledTime>2023-08-15T22:53:00Z</TimetabledTime>
+									</ServiceDeparture>
+									<Order>5</Order>
+								</LegIntermediate>
+								<LegIntermediate>
+									<siri:StopPointRef>6</siri:StopPointRef>
+									<StopPointName>
+										<Text xml:lang="de">Regensburg Hbf</Text>
+									</StopPointName>
+									<ServiceArrival>
+										<TimetabledTime>2023-08-15T23:54:00Z</TimetabledTime>
+									</ServiceArrival>
+									<ServiceDeparture>
+										<TimetabledTime>2023-08-15T23:56:00Z</TimetabledTime>
+									</ServiceDeparture>
+									<Order>6</Order>
+								</LegIntermediate>
+								<LegIntermediate>
+									<siri:StopPointRef>7</siri:StopPointRef>
+									<StopPointName>
+										<Text xml:lang="de">Nürnberg Hbf</Text>
+									</StopPointName>
+									<ServiceArrival>
+										<TimetabledTime>2023-08-16T00:56:00Z</TimetabledTime>
+									</ServiceArrival>
+									<ServiceDeparture>
+										<TimetabledTime>2023-08-15T01:51:00Z</TimetabledTime>
+									</ServiceDeparture>
+									<Order>7</Order>
+								</LegIntermediate>
+								<!-- some stops omitted -->
+								<LegAlight>
+									<siri:StopPointRef>20</siri:StopPointRef>
+									<StopPointName>
+										<Text xml:lang="de">Hamburg Altona</Text>
+									</StopPointName>
+									<ServiceArrival>
+										<TimetabledTime>2023-08-16T09:03:00Z</TimetabledTime>
+									</ServiceArrival>
+									<Order>12</Order>
+								</LegAlight>
+								<Service>
+									<OperatingDayRef>2023-08-15</OperatingDayRef>
+									<JourneyRef>NJ-OeBB-1230123-123111</JourneyRef>
+									<siri:LineRef>OeBB:NJ12311</siri:LineRef>
+									<siri:DirectionRef>H</siri:DirectionRef>
+									<Mode>
+										<PtMode>rail</PtMode>
+										<siri:RailSubmode>nightRail</siri:RailSubmode>
+										<Name>
+											<Text xml:lang="de">Zug</Text>
+										</Name>
+										<ShortName>
+											<Text xml:lang="de">NJ</Text>
+										</ShortName>
+									</Mode>
+									<PublishedServiceName>
+										<Text xml:lang="de">NJ 490</Text>
+										<!-- the PublishedService changes in the middle, therefore having this PublishedServiceName and the train number
```

Comment:
What does "the PublishedService changes in the middle" mean?

Reply:
changed